mmv1/third_party/terraform/resources/resource_compute_instance.go

```diff
@@ -641,6 +641,29 @@ func resourceComputeInstance() *schema.Resource {
 					},
 				},
 			},
+			"advanced_machine_features": {
```

Comment:
If we add AtLeastOneOf here, we should probably also add it to the template to keep them in sync with similar diff/recreate behavior.
https://github.com/GoogleCloudPlatform/magic-modules/pull/4850

Reply:
AtLeastOneOf might also resolve this issue: https://github.com/hashicorp/terraform-provider-google/issues/9436
But it would make the block required for users to set, right?

Reply:
My intention with AtLeastOneOf is to apply it to `enable_nested_virtualization` and `threads_per_core`, so that if `advanced_machine_features` is specified at all, one of those fields will be required.